### ISSUE TITLE
How to delete entries?

### ISSUE THREAD

**Issue by Gnopps** [mgmeyers/obsidian-list-callouts#53]
I've added some callouts but I can't see any way to delete them. As a workaround I set the character as something that won't be used. Is is possible to delete a callout configuration?

![image](https://github.com/mgmeyers/obsidian-list-callouts/assets/16313148/2bddf0ff-16b4-4f0e-8f10-454e6d0145f6)

**Comment by ryanjamurphy** [comment]
You cannot delete the defaults at the moment, only list-callouts that you've added yourself:

![CleanShot 2024-03-06 at 11 36 41](https://github.com/mgmeyers/obsidian-list-callouts/assets/3618647/27c203b8-17b3-4e09-8ce6-0fee355a85fb)

**Comment by dimateos** [comment]
Yeah, also requested in https://github.com/mgmeyers/obsidian-list-callouts/issues/34.
Would be nice to be able to edit them as regular ones + the option to reset settings.

**Comment by danyim** [comment]
I recently created a fork called "List Callouts, Improved" (https://github.com/danyim/obsidian-list-callouts-improved) since this project looks abandoned. It fixes exactly this — you can delete any callout entry, including the defaults, from the settings.
